nss-day-cohort-30/bangazon-site-cunning-carps
# Abbey's Ticket Ten Branch

#Description
Given a user wants to see if there are erroneous orders in the system
When the user clicks on the Reports item in the navigation bar
Then a hyperlink labeled Multiple Orders should be in a list of reports to run

Given user is viewing the list of reports to run
When the Multiple Orders hyperlink is clicked
Then the user should be presented with a list of customer names and order counts where each customer has more than one open order in the system.

#Type of change
New feature (non-breaking change which adds functionality)

 
#Testing Instructions for Change Made
git fetch --all
git checkout abbeyticketten
dotnet run
verify...

Update Database before testing code 

#Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
 I have added test instructions that prove my fix is effective or that my feature works
